src/Controller/AddOnOptionsController.py

```diff
@@ -4,16 +4,15 @@
 #                                                                                                    #
 ######################################################################################################
 import csv
-import sys
 import webbrowser
 from collections import deque
 
 from PyQt5.QtWidgets import QFileDialog
 
-from src.data.csv import *
 from src.View.AddOnOptions import *
```

Comment:
best to eliminate any import * and make it explicit so that we know what comes from where, and to avoid function or class name clashes